### PR TITLE
waf: allow to run wrapper on windows

### PR DESCRIPTION
**Pull request by lucasdemarchi**

### CODE REVIEW

File: waf [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 import os.path as p
 import sys
 
 d = p.dirname(p.realpath(__file__))
 try:
-    os.execv(os.path.join(d, 'modules/waf/waf-light'), sys.argv)
-except OSError:
+    subprocess.check_call(['python', p.join(d, 'modules', 'waf', 'waf-light')] + sys.argv[1:])
+except subprocess.CalledProcessError:
     print('Missing waf submodule. Trying to get it')
```
Comment (guludo) [inline waf]: Hm, wouldn't this catch configure and build errors as well?

Comment (guludo) [inline waf]: I think we should verify the existence of `modules/waf/waf-light` before running it.

Comment (lucasdemarchi) [inline waf]: > Hm, wouldn't this catch configure and build errors as well?

humn... right. Before it was a more specific OSError.

Comment (lucasdemarchi) [inline waf]: > I think we should verify the existence of modules/waf/waf-light before running it.

it's a TOCTOU, but maybe it's fine

**Comment by lucasdemarchi** [comment]
New version, tested on Linux and Windows. Working fine for me.

**Comment by guludo** [comment]
lgtm

**Comment by lucasdemarchi** [comment]
Applied, thanks!

**Pull request closed without merging**